en/docs/portal/transports/show-history.md

```diff
@@ -1,10 +1,18 @@
 # Show History of a Transport
+1. Click **Transport** in the left pane to display the Transport page.
+2. Click **Action** button on the transport you want to show history and select **Show History**. If the transport is not in the current page, you can search for it in the search box.
+![null](</docs/resources/images/transports/show-history-1.png>)
 
-The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of the HDT portal.
-To view the history of a transport, click the "Action" button of the transport entry in the list and select "Show History". If the transport is not in the current page, you can search for it in the search box.
+## Compare Selected Versions
+1. In the prompt Change History window, select the versions you would like to compare and click **Compare Selected Versions**.
+![null](</docs/resources/images/transports/show-history-2.png>)
 
-![null](</docs/resources/images/transports/show-history-1.png>)
+2. In the prompt Compare window, you can compare the configurations between the versions you picked from the versions drop-down box. The differences are highlighted in color.
+![null](</docs/resources/images/transports/show-history-3.png>)
 
-You can then compare the selected versions and roll back to a previous version.
+## Roll back
+1. In the prompt Change History window, select the versions you would like to roll back to and click **RollBack**.
+![null](</docs/resources/images/transports/show-history-4.png>)
 
-![null](</docs/resources/images/transports/show-history-2.png>)
+2. In the prompt Compare window, Click **&check; RollBack**.
```

Comment:
remove the word "prompt" and change "RollBack" to "Rollback" to match your portal.